doc/fastcat_device_config_parameters.md

```diff
@@ -1137,3 +1137,50 @@ This example implements an absolute value function over the range of [-9, 9]
   - observed_device_name: sig_gen_1
     request_signal_name:  output
 ```
+
+
+## ThreeNodeThermalModel
+
+| Parameter   | Description |
+| ----------- | ----------- |
+| thermal_mass_node_1      | The thermal mass that represents the winding node (node 1) |
+| thermal_mass_node_2      | The thermal mass that represents the stator node (node 2) |
+| thermal_res_nodes_1_to_2      | The effective thermal resistance between nodes 1 and 2 |
+| thermal_res_nodes_2_to_3      | The effective thermal resistance between nodes 2 and 3 |
+| winding_res      | The electrical resistance of the motor windings at 20 deg C |
```

Comment:
Can we separate this into two parameters with one being the resistance at a reference temp. and the other parameter being the reference temp.
If this is to be used outside, it is pretty restricting to have the resistance at 20 deg C specifically

Reply:
yeah, this is a good question -- so in the implementation, this reference value here is supplied and using the measured temp and the coefficient-or-resistance for thermal change, the winding resistance is recalculated for the current temp. That would be what is used -- I believe 20 deg C is the de-facto calibration temp for most of these values though so that's why it's specified like that.

Reply:
Because we have the temperature as a constant, it might be worth having a default value for he reference temp. of 20 and then have the ability to change this.
I just did a quick google and the reference temperature for a PTC thermistor can sometimes be 25 deg C:
https://www.tdk-electronics.tdk.com/download/531162/d78540dfe0589d2bd90cabef477c90b9/pdf-general-technical-information.pdf

Reply:
I can just remove any comments to 20 deg specifically, and that should clear up any potential miscommunication

Reply:
As the code is written, the comment should be kept because we hard-code the REFERENCE_TEMPERATURE constant at 20.0. If others use a resistance value taken at a different temperature, the math will be incorrect.
It's up to you if you want users to have the flexibility to have resistance values at different reference temperatures.

Reply:
Ahh, I see what you referring to. I wasn't thinking about the const value set in code since this comment was on the markdown file haha. 

yeah, I think it's good to make the ref temp a parameter here, so I'll add that